ydb/docs/en/core/reference/ydb-cli/sql.md

```diff
@@ -0,0 +1,120 @@
+# Query execution
+
+You can use the `{{ ydb-cli }} sql` subcommand to execute any SQL query. The query can be of any type (DDL, DML e.t.c.) and can consist of several sub-queries. The `{{ ydb-cli }} sql` subcommand establishes a streaming connection and retrieves data through it. With the in-stream query execution, no limit is imposed on the amount of data read.
+
+General format of the command:
+
+```bash
+{{ ydb-cli }} [global options...] sql [options...]
+```
+
+* `global options`: [Global parameters](commands/global-options.md).
+* `options`: [Parameters of the subcommand](#options).
+
+View the description of this command by calling it with `--help` option:
+
+```bash
+{{ ydb-cli }} sql --help
+```
+
+## Parameters of the subcommand {#options}
+
+#|
+|| Name | Description ||
+|| `-h`, `--help` | Print general usage help ||
+|| `-hh` | Print complete usage help. Prints some specific options that are not printed with `--help` ||
+|| `-s`, `--script` | Script (query) text to execute. ||
+|| `-f`, `--file` | Path to a file with query text to execute. Path `-` means reading query text from `stdin`. ||
+|| `--stats` | Statistics mode.<br/>Available options:<br/>
+<ul>
+<li>`none` (default): Do not collect.</li>
+<li>`basic`: Collect aggregated statistics for updates and deletes per table.</li>
+<li>`full`: Add execution statistics and plan on top of `basic`.</li>
+<li>`profile`: Collect detailed execution statistics including statistics for individual tasks and channels.</li>
+</ul> ||
+|| `--explain` | Execute explain request for the query. Shows query logical plan. The query is not actually executed, thus does not affect database data. ||
+|| `--explain-ast` | Same as `--explain`, but in addition to the query logical plan, you an AST (abstract syntax tree) is printed. The AST section contains a representation in the internal miniKQL language. ||
```

Comment:
Make miniKQL to be a link to the glossary (add an entry if it's missing). Make AST to be the link to Wikipedia.